### DISCUSSION TITLE
Why `updateDaysBySectionInWeekView` is deprecated?

### DISCUSSION THREAD

**Discussion by mohtasimabrar** [kvyatkovskys/KVKCalendar#301] [Q&A]
I saw that in CalendarView+Extension, `updateDaysBySectionInWeekView` is deprecated. But according to this [change](https://github.com/kvyatkovskys/KVKCalendar/compare/0.6.5...0.6.6#diff-9f0d4365ec8aef648282bbd4e479a6386210fbba801ad4298e040d3c5a3aa5dcR344), this seems to be an improvement. 
Can you kindly explain the case a bit and if it is indeed deprecated, which method should I use instead?

**Comment by kvyatkovskys** [comment]
Hello!
That's right. You need to use the `func updateStyle(:)` and work with this `style.week.daysInOneWeek` property.

**Reply by mohtasimabrar** [reply]
Hello!
Thanks for getting back. I had tried using `func updateStyle(:)` but unfortunately it didn't work like `updateDaysBySectionInWeekView` . 
`updateDaysBySectionInWeekView` has 3 methods called inside it, one of them is `weekView.updateScrollableWeeks()` which is not called inside `func updateStyle(:)` . Can you kindly let me know how I can get `weekView.updateScrollableWeeks()` called?

**Reply by kvyatkovskys** [reply]
For the quick solution 🤔 try to call `.reloadData()`
